src/java.base/share/classes/javax/security/auth/Subject.java

```diff
@@ -205,16 +205,19 @@ public Subject() {
     public Subject(boolean readOnly, Set<? extends Principal> principals,
                    Set<?> pubCredentials, Set<?> privCredentials)
     {
-        collectionNullClean(principals);
-        collectionNullClean(pubCredentials);
-        collectionNullClean(privCredentials);
+        LinkedList<Principal> princList
```

Comment:
Suggest principleList

Reply:
Accepted. The "c" there has a weird pronounciation.